python/lsst/ctrl/bps/wms/panda/cmd_line_embedder.py

```diff
@@ -0,0 +1,94 @@
+import os
+
+
+class CommandLineEmbedder:
+    """
+    Class embeds static (constant across a task) values into the pipeline execution command line

```

Comment:
This seems like something that is completely generic and not panda-specific at all. Is the issue here that BPS is using a default syntax that htcondor understands internally but which panda needs to have code to expand? Either way this surely should be in a generic part of bps so that other plugins can use it. Is there panda-specific code here?

Reply:
See comment below about <ENV syntax being plugin specific.  Also, currently the HTCondor/Pegasus plugins don't care specifically about the ids vs other values.  We could have the code that builds the GenericWorkflow ensure that only variables that have different values within a workflow are left, but there hasn't been any request to do so.